### PR TITLE
[ApplicationRequest] Some improvements & Twig update

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Entity/Adherent.php [hunk 1]
```diff
@@ -1102,8 +1102,7 @@ public function setReferent(array $tags, string $markerLatitude = null, string $
 
     public function isReferent(): bool
     {
-        return $this->managedArea instanceof ReferentManagedArea
-            && !$this->managedArea->getTags()->isEmpty();
+        return $this->managedArea instanceof ReferentManagedArea;
```
Comment (Remg) [inline src/Entity/Adherent.php]: i remember, a while ago, this extra check missing made the application send an email to the whole database :)
are we sure we want to remove this check ?
is it for performance?
what if we forget to apply this eventlistener on a form one day?

File: templates/admin/application_request/_favorite_cities.html.twig [hunk 2]
```diff
@@ -1,10 +1,5 @@
 {% extends 'SonataAdminBundle:CRUD:base_list_field.html.twig' %}
 
 {% block field %}
-    {% if object.favoriteCities %}
-        {% for city in object.favoriteCities %}
-            {{ get_city_name_from_insee_code(city) }} ({{ city }})
-            {% if not loop.last %}, {% endif %}
-        {% endfor %}
-    {% endif %}
+    {{ object.favoriteCities|map(code => get_city_name_from_insee_code(code))|join(', ') }}
```
Comment (Remg) [inline templates/admin/application_request/_favorite_cities.html.twig]: 👍

File: src/Entity/Adherent.php (hunk 1 above)
Comment (Remg) [reply src/Entity/Adherent.php]: same modification/questions would apply to : 
https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L1149
https://github.com/EnMarche/en-marche.fr/blob/master/src/Entity/Adherent.php#L1151

etc etc

i mostly see these checks as a security if we ever have a corrupted entry in the database.

Comment (ottaviano) [reply src/Entity/Adherent.php]: agree, I prefer to rollback this part

**Pull request merged**